tests/MyGame/Example/MonsterStorage_grpc.go

```diff
@@ -4,19 +4,16 @@
 
```

Comment:
This file is auto generated, so any changes here should be because of code generation, not gofmt.